user/pages/03.commerce2/01.user-guide/10.commerce-pos/docs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: POS
+taxonomy:
+    category: docs
+---
+
+! We need help filling out this section! Feel free to follow the *edit this page* link and contribute.
+
+POS is enabled in Drupal Commerce 2.x with an external module
+[Commerce POS](https://drupal.org/project/commerce_pos). This
+module provides a Point of Sale interface for Drupal Commerce,
```

Comment:
Should we include a link to wikipedia in case someone is unsure what "Point of Sale" is?